dispatchrouter/src/main/fabric8/dispatch-router-route.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Route
+metadata:
+  labels:
+    app: hono
+    name: ${project.artifactId}
+  name: ${project.artifactId}
+spec:
+  to:
+    kind: Service
+    name: ${project.artifactId}
+  port:
+    targetPort: 5671
```

Comment:
the dispatch router deployment exposes port 5671 and 5672. With Docker compose you need to map them to 15671 and 15672 maybe because you are on localhost and they are already assigned to hono server (5671) and broker (5672). It's different here for the OpenShift deployment.